main/ZmqttDiscovery.ino

```diff
@@ -245,19 +245,27 @@ void createDiscovery(const char* sensor_type,
     sensor["stat_t"] = state_topic;
   }
 
+  // We check if the class belongs to HAAS class list
+  bool HASSClass = false;
+  for (int i = 0; i < 14; i++) { // see class list and size into config_mqttDiscovery.h
```

Comment:
If the change mentioned below is accepted this will need to be changed to accommodate it to:
```
int num_classes = sizeof(availableHASSClasses) / sizeof(availableHASSClasses[0]);
for (int i = 0; i < num_classes; i++) {
```